packages/optimistic-oracle/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
```

Comment:
@mrice32 is this a reasonable starting change log for this package?

Reply:
Yeah, that works. If you want, you can add a line for the initial version -- not sure what we've done for the other packages.

Reply:
done